app/src/main/res/values-v21/styles_congress.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <style name="PopupMenuOverflowStyle" parent="@style/Widget.AppCompat.Light.PopupMenu.Overflow">
```

Comment:
Should all non-affected platform versions use the default style again? That is, should there be a `res/values-v22/styles_congress.xml`?

Reply:
I am not sure if I understand. `PopupMenuOverflowStyle` is also defined in `values/styles_congress.xml`. This should overwrite whatever is defined in `values-v21` on devices running Android 22 and above.

Reply:
`values-v21` is used on API 21+ devices unless there's a `values-vX` folder more closely matching the device API level you're running on. 
You can observe this by replacing the color used in `drawable-v21/popup_background.xml` with something very distinct. I'll show up on your API 29 device.

Reply:
Thank you.
:heavy_check_mark: Fixed in 70edb20497508fd918a7ec404fa9d5c0b4319b60.

Reply:
@cketti You are welcome to merge if everything is resolved from your point of view.